lib/model/game_state.dart

```diff
@@ -79,4 +79,9 @@ class GameState with ChangeNotifier {
 
     notifyListeners();
   }
+
+  void endGame(GameResult result) {
+    this.result = result;
+    notifyListeners();
+  }
```

Comment:
Looks good - we could add to the `state methods should notifier listeners` test to cover this off (and also rename it maybe because that is some bodged grammar!)